doc/rear.8

```diff
@@ -31,7 +31,7 @@
 rear \- bare metal disaster recovery and system migration tool
 .SH "SYNOPSIS"
 .sp
-\fBrear\fR [\fB\-h\fR|\fB\-\-help\fR] [\fB\-V\fR|\fB\-\-version\fR] [\fB\-dsSv\fR] [\fB\-D\fR|\fB\-\-debugscripts\fR \fISET\fR] [\fB\-c\fR \fIDIR\fR] [\fB\-C\fR \fICONFIG\fR] [\fB\-r\fR \fIKERNEL\fR] [\-\-] \fICOMMAND\fR [\fIARGS\fR\&...]
+\fBrear\fR [\fB\-h\fR|\fB\-\-help\fR] [\fB\-V\fR|\fB\-\-version\fR] [\fB\-dsSv\fR] [\fB\-D\fR|\fB\-\-debugscripts\fR \fISET\fR] [\fB\-c\fR \fIDIR\fR] [\fB\-C\fR \fICONFIG\fR] [\fB\-r\fR \fIKERNEL\fR] [\fB\-n\fR|\fB\-\-non\-interactive\fR] [\fB\-e\fR|\fB\-\-expose\-secrets\fR] [\-\-] \fICOMMAND\fR [\fIARGS\fR\&...]
```

Comment:
BTW, this shows that after adding the `-e` switch nobody ran `make doc` :shrug:

Reply:
Yes @schlomo good observation, so I don't object to committing this file after you have regenerated it, despite having seemingly unrelated changes.
I hate having this autogenerated file in the source tree btw. As there is a Make target to generate it, it should be generated during build, IMO. Its presence in sources makes the build undeterministic: if the Git checkout has older rear.8 than rear.adoc, it will get regenerated during the build, but if it is newer, it will not be generated (masking possible errors). But this discussion belongs elsewhere.

Reply:
I can promise to amend our `.gitignore` and remove all build results after this PR is done.